test/functional/reproducibleCompare/playlist.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>Reproducible_Comparison</testCaseName>
```

Comment:
I would like us to change the name of this test to be more descriptive, something like "Rebuild_Same_JDK_Reproducibility_Test", this will allow us to add other similar tests in the future, such as "Different_Vendor_JDKs_Comparibility_Test" and removing the word 'Comparison' from the name may help reduce confusion.

Reply:
Updated.